### ISSUE TITLE
content generation still failing

### ISSUE THREAD

**Issue by terminills** [terminills/gator#241]
content generation is still failing even though I can manually generate on this server

python ../demo_ai_video_generation.py

======================================================================
🎥 GATOR AI-POWERED VIDEO GENERATION DEMO
Enhanced Video Features with Real Image Generation
======================================================================

======================================================================
🎬 DEMO: Placeholder vs AI-Generated Video Frames
======================================================================

1. Initializing AI Model Manager...
   - GPU Type: rocm
   - GPU Memory: 59.96875 GB
   - Available image models: 3

----------------------------------------------------------------------
2. Generating video with PLACEHOLDER frames (fast mode)
----------------------------------------------------------------------
   Prompts: ['A serene mountain landscape at sunrise', 'A bustling city skyline at night']
   Quality: STANDARD (1280x720)
   Use AI: False

✅ Placeholder video generated!
   📁 Path: generated_content/videos/video_multi_frame_20251112_141835.mp4
   ⏱️  Duration: 4.0 seconds
   📊 File size: 46.08 KB
   🎞️  Scenes: 2

----------------------------------------------------------------------
3. Generating video with AI-GENERATED frames (quality mode)
----------------------------------------------------------------------
   Prompts: ['A serene mountain landscape at sunrise', 'A bustling city skyline at night']
   Quality: STANDARD (1280x720)
   Use AI: True
   Note: AI generation may take several minutes for first run
         (model download + generation time)
Keyword arguments {'safety_checker': None, 'requires_safety_checker': False} are not expected by StableDiffusionXLPipeline and will be ignored.
Loading pipeline components...:   0%|                                                                                                             | 0/5 [00:00<?, ?it/s]`torch_dtype` is deprecated! Use `dtype` instead!
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00, 10.66it/s]
xformers not available, using default attention
Diffusers generation failed: 'NoneType' object has no attribute 'tokenize'
❌ IMAGE GENERATION FAILED after 9.50s
   Error: 'NoneType' object has no attribute 'tokenize'
   Model attempted: sdxl-1.0
AI frame generation failed: 'NoneType' object has no attribute 'tokenize', using placeholder
Diffusers generation failed: 'NoneType' object has no attribute 'tokenize'
❌ IMAGE GENERATION FAILED after 0.02s
   Error: 'NoneType' object has no attribute 'tokenize'
   Model attempted: sdxl-1.0
AI frame generation failed: 'NoneType' object has no attribute 'tokenize', using placeholder

✅ AI-generated video created!
   📁 Path: generated_content/videos/video_multi_frame_20251112_141846.mp4
   ⏱️  Duration: 4.0 seconds
   📊 File size: 46.08 KB
   🎞️  Scenes: 2

   💡 Compare the two videos to see the difference!
      Placeholder: Simple gradients with text
      AI-generated: Actual images from prompts

======================================================================
🎬 DEMO: Direct VideoProcessingService with AI
======================================================================

1. Initializing services...

2. Generating multi-scene video with AI...
   Scenes: 3
   Quality: HIGH (1920x1080)
   Transition: CROSSFADE
Keyword arguments {'safety_checker': None, 'requires_safety_checker': False} are not expected by StableDiffusionXLPipeline and will be ignored.
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  6.97it/s]
xformers not available, using default attention
Diffusers generation failed: 'NoneType' object has no attribute 'tokenize'
❌ IMAGE GENERATION FAILED after 5.74s
   Error: 'NoneType' object has no attribute 'tokenize'
   Model attempted: sdxl-1.0
AI frame generation failed: 'NoneType' object has no attribute 'tokenize', using placeholder
Diffusers generation failed: 'NoneType' object has no attribute 'tokenize'
❌ IMAGE GENERATION FAILED after 0.02s
   Error: 'NoneType' object has no attribute 'tokenize'
   Model attempted: sdxl-1.0
AI frame generation failed: 'NoneType' object has no attribute 'tokenize', using placeholder
Diffusers generation failed: 'NoneType' object has no attribute 'tokenize'
❌ IMAGE GENERATION FAILED after 0.02s
   Error: 'NoneType' object has no attribute 'tokenize'
   Model attempted: sdxl-1.0
AI frame generation failed: 'NoneType' object has no attribute 'tokenize', using placeholder

✅ Video generated with AI frames!
   📁 Path: /tmp/gator_ai_videos/video_multi_frame_20251112_141853.mp4
   ⏱️  Total duration: 6.0 seconds
   📐 Resolution: 1920x1080
   🎞️  Scenes: 3
   🔀 Transitions: crossfade
   📊 File size: 73.11 KB

======================================================================
✅ ALL DEMOS COMPLETED!
======================================================================

📁 Generated videos are in:
   - AI Manager: generated_content/videos/
   - Direct service: /tmp/gator_ai_videos/

📝 Key Points:
   - use_ai_generation=False: Fast placeholder frames
   - use_ai_generation=True: Real AI-generated images
   - Requires GPU or API keys for AI generation
   - Fallback to placeholder if AI generation fails

🧪 Run tests:
   pytest tests/unit/test_ai_video_frame_generation.py -v

(torch_rocm7) terminills@terminills-05:~/Desktop/gator/src$ python3 - <<'EOF'
from diffusers import StableDiffusionXLPipeline; import torch
pipe = StableDiffusionXLPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, variant="fp16", use_safetensors=True).to("cuda") 
print(pipe("a test image", num_inference_steps=5).images[0])
EOF
Loading pipeline components...:  29%|████████████████████████████▊                                                                        | 2/7 [00:00<00:00, 13.48it/s]`torch_dtype` is deprecated! Use `dtype` instead!
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 7/7 [00:00<00:00,  9.46it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:05<00:00,  1.13s/it]
<PIL.Image.Image image mode=RGB size=1024x1024 at 0x7DEDC3826CB0>
